docs/source/data_ingestion_and_preparation/connectors/duckdb.md

```diff
@@ -0,0 +1,58 @@
+# DuckDB
+
+[DuckDB](https://duckdb.org/) is an in-process analytical database that uses a SQL dialect and can read data from CSV, Parquet and JSON file formats. It requires no external dependencies and has the ability to run in-process on its host application. This simplicity in deployment makes it a good choice for lightweight operations when using Sycamore for development work or with low data loads.
```

Comment:
Remove "can read data from..." given we are just talking about loading from Sycamore